src/brevitas/graph/equalize.py

```diff
@@ -166,8 +225,16 @@ def _equalize(model, regions, iterations):
         if name in name_set:
             name_to_module[name] = module
     for i in range(iterations):
+        print(f"Equalize iter {i}")
+        scale_factor_max = torch.tensor(0.0)
         for region in regions:
-            _cross_layer_equalization([name_to_module[n] for n in region[0]], [name_to_module[n] for n in region[1]])
+            scale_factors_region = _cross_layer_equalization([name_to_module[n] for n in region[0]], [name_to_module[n] for n in region[1]], merge_bias, bias_shrinkage)
+            scale_factor_region_max = torch.max(torch.abs(1-scale_factors_region))
+            scale_factor_max = torch.max(scale_factor_max, scale_factor_region_max)
+        if threshold is not None:
+            if scale_factor_max < threshold:
+                print("Treshold reached, early stopping")
```

Comment:
No prints in the codebase unless it's something that a user can easily remove. If anything, we should add support for a proper logger